tools/smina/smina.xml

```diff
@@ -0,0 +1,364 @@
+      --atom_terms                 optionally write per-atom interaction term values
+      --atom_term_data             embedded per-atom interaction terms in output sd data
+    
+    **Scoring and minimization options:**
+      --scoring arg                specify alternative builtin scoring function
+      --custom_scoring arg         custom scoring function file
+      --custom_atoms arg           custom atom type parameters file
+      --score_only                 score provided ligand pose
+      --local_only                 local search only using autobox (you probably 
+                                   want to use --minimize)
+      --minimize                   energy minimization
+      --randomize_only             generate random poses, attempting to avoid 
+                                   clashes
+      --minimize_iters arg         number iterations of steepest descent; default 
+                                   scales with rotors and usually isn't sufficient 
+                                   for convergence
+      --accurate_line              use accurate line search
+      --minimize_early_term        stop minimization before convergence conditions 
+                                   are fully met
+      --approximation arg          approximation (linear, spline, or exact) to use
+      --factor arg                 approximation factor: higher results in a 
+                                   finer-grained approximation
+      --force_cap arg              max allowed force; lower values more gently 
+                                   minimize clashing structures
+      --user_grid arg              Autodock map file for user grid data based 
+                                   calculations
+      --user_grid_lambda arg       scales user_grid and functional scoring
+      --print_terms                print all available terms with default 
+                                   parameterizations
+      --print_atom_types           print all available atom types
+    
+    **Misc (optional):**
+      --cpu arg                  the number of CPUs to use (the default is to try 
+                                 to detect the number of CPUs or, failing that, use
+                                 1)
+      --seed arg                 explicit random seed
+      --exhaustiveness arg       exhaustiveness of the global search (roughly 
+                                 proportional to time)
+      --num_modes arg            maximum number of binding modes to generate
+      --energy_range arg         maximum energy difference between the best binding
+                                 mode and the worst one displayed (kcal/mol)
+      --min_rmsd_filter arg      rmsd value used to filter final poses to remove 
+                                 redundancy
+      --addH arg                 automatically add hydrogens in ligands (on by 
+                                 default)
```

Comment:
I'd remove all the copy-pasted command line help, everything should be already in the param help attributes.